src/react/components/pages/prebuiltPredict/prebuiltPredictPage.tsx

```diff
@@ -272,14 +272,16 @@ export class PrebuiltPredictPage extends React.Component<IPrebuiltPredictPagePro
                             </div>
                         </div>
                         <Separator className="separator-right-pane-main">2. Get prediction.</Separator>
-                        <div className="p-3" style={{ marginTop: "8px" }}>
-                            <Toggle theme={getLightGreyTheme()}
-                                className="predict-mode-toggle"
-                                defaultChecked
-                                onText="Call live service"
-                                offText="Use predicted file"
-                                onChange={this.handleLiveModeToggleChange} />
-                        </div>
+                        {process.env.NODE_ENV === "development" &&
```

Comment:
suggestion: This would not 100% suit our needs, per my understanding our webapp all use the same docker file/image/configs/mode, which means our testing environments will hide this toggle. 
IMO we might follow the same flow of `enableAPIVersionSelection `and config it in various branch.